app/models/car.rb

```diff
@@ -0,0 +1,6 @@
+# app/models/car.rb
+class Car < ApplicationRecord
+    # Attributes
+    attr_accessor :model, :color, :image_url, :price
```

Comment:
Here adding `attr_accessor` is not necessary, because Rails does it automatically, you can refer to this [link](https://stackoverflow.com/questions/67758670/rails-attr-accessor-association) for more info about this.